src/catchup/test/CatchupWorkTests.cpp

```diff
@@ -17,6 +17,11 @@ using namespace stellar;
 auto max = std::numeric_limits<uint32_t>::max();
 namespace stellar
 {
+
+std::random_device gRandomDevice;
```

Comment:
We recently consolidated random_device / random_engine usage so that all tests would be deterministically seedable. Please use extern std::default_random_engine gRandomEngine defined in util/Math.h rather than establishing your own elsewhere.